src/python/tests/core/bot/fuzzers/libFuzzer/launcher_test_data/oom_in_seed_corpus.txt

```diff
@@ -0,0 +1,127 @@
+INFO:        0 files found in /mnt/scratch0/clusterfuzz/bot/inputs/fuzzer-testcases-disk/temp-55482/new
+INFO:    14330 files found in /mnt/scratch0/clusterfuzz/bot/inputs/data-bundles/global/origin_trial_token_fuzzer
+INFO:     5000 files found in /mnt/scratch0/clusterfuzz/bot/inputs/fuzzer-testcases-disk/temp-55482/mutations
+INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 1048575 bytes
+INFO: seed corpus: files: 19330 min: 1b max: 1048575b total: 26242064b rss: 1150Mb
+#128	pulse  cov: 788 ft: 1541 corp: 92/157b exec/s: 64 rss: 1153Mb
+#256	pulse  cov: 948 ft: 2285 corp: 182/404b exec/s: 64 rss: 1156Mb
+#512	pulse  cov: 1204 ft: 3767 corp: 348/998b exec/s: 73 rss: 1161Mb
+#1024	pulse  cov: 1386 ft: 5453 corp: 697/2646b exec/s: 73 rss: 1179Mb
+#2048	pulse  cov: 1632 ft: 8051 corp: 1342/7769b exec/s: 73 rss: 1231Mb
+#4096	pulse  cov: 3183 ft: 14143 corp: 2253/21Kb exec/s: 80 rss: 1326Mb
+#8192	pulse  cov: 3637 ft: 20165 corp: 4125/63Kb exec/s: 80 rss: 1594Mb
+#16384	pulse  cov: 3894 ft: 28749 corp: 7462/307Kb exec/s: 79 rss: 1881Mb
+#19333	INITED cov: 3912 ft: 31745 corp: 8755/8252Kb exec/s: 74 rss: 2053Mb
```

Comment:
Before fix, we were doing

edge_coverage=3894
initial_edge_coverage=3912
new_edges = 3894-3912= negative.